site/src/pages/release-notes/1.24.3.mdx

```diff
@@ -0,0 +1,27 @@
+---
+date: 2023-07-25
+---
+
+## 🛠️ Bug fixes
+
+- Armeria server now strips [matrix variables](https://docs.spring.io/spring-framework/reference/web/webmvc/mvc-controller/ann-methods/matrix-variables.html)
```

Comment:
Should we add a separate section like ':detective: Security'?

Reply:
I should've put this in the security section. 😓